### PR TITLE
2.0-preview

### PR DESCRIPTION
**Pull request by e111077**
Requesting creation of a `2.0-preview` branch on GoogleWebComponents/google-apis so I can merge with that instead of master. 

Polymer 2.0 preview. Main changes, dependencies are 2.0 hybrid elements and `this.fire` elements have the `composed: true` flag for bubbling past shadow roots.

working on this to fix incompatibilities with `google-youtube` which is blocking the `2.0-preview` branch of `app-route`.

### CODE REVIEW

**Comment by Scarygami** [comment]
Hope this gets merged soon, since it's blocking the upgrade of several other components.

Polymer dependency should be changed to `"Polymer/polymer#^2.0.0-rc.1"` now though.

File: demo/index.html [hunk 1]
```diff
@@ -10,38 +10,43 @@
...
+    </script>
+  </dom-module>
 
-    t.loaded = function(e) {
-      document.querySelector('#messages').innerHTML +=
-        e.target.localName + ' loaded' + '<br>';
-      console.log(e.target.localName + ' loaded', event.target.api);
-    };
-  </script>
+  <demo-element></demo-element>
```
Comment (ebidel) [inline demo/index.html]: Can we keep the light dom. We typically like to show more usage of how to use the element from "outside": of polymer. Granted, it was using dom-bind :)

**Comment by ebidel** [comment]
Sorry! I didn't realize this was waiting on me. 

@e111077 I invited you to the org. You should be able to create the `2.0-preview` branch.

LGTM, just the one nit.

File: demo/index.html (hunk 1 above)
Comment (e111077) [inline demo/index.html]: okey dokey

**Pull request merged**